fission/portforward.go

```diff
@@ -68,12 +68,18 @@ func runPortForward(kubeConfig string, labelSelector string, localPort string, f
 
 	// make a useful error message if there is more than one install
 	if len(podList.Items) > 1 {
-		namespaces := make([]string, 0)
+		namespaces := make(map[string]struct{})
+		nsList := make([]string, 0)
 		for _, p := range podList.Items {
-			namespaces = append(namespaces, p.Namespace)
+			if _, ok := namespaces[p.Namespace]; !ok {
+				namespaces[p.Namespace] = struct{}{}
+				nsList = append(nsList, p.Namespace)
+			}
+		}
+		if len(nsList) > 1 {
```

Comment:
Don't you have to handle the case where there are more pods but only one namespace?  For some reason?  In that case you have to talk to any pod that's in a healthy state...

Reply:
Thank for pointing that out! Just push a new commit to fix the problem.